custom/icds_reports/utils/__init__.py

```diff
@@ -385,6 +385,16 @@ def get_anamic_status(value):
     else:
         return DATA_NOT_ENTERED
 
+def get_anemic_status(value):
+    if value['anemic_severe']:
+        return 'Severe'
```

Comment:
I'm assuming these values don't need to be translated?

Reply:
Yeah, all these strings for now are untranslated